packages/web-runtime/tests/unit/composables/upload/useUpload.spec.ts

```diff
@@ -10,6 +10,15 @@ import { SpaceResource } from 'web-client/src/helpers'
 import { mock } from 'jest-mock-extended'
 
 describe('useUpload', () => {
+  beforeEach(() => {
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    jest.spyOn(console, 'error').mockImplementation(() => {})
```

Comment:
```suggestion
    jest.spyOn(console, 'error').mockImplementation(() => undefined)
```

Then you don't need the eslint ignore